### ISSUE TITLE
Detail syndication requirements and workflow in dev docs #docs

### ISSUE THREAD

**Issue by mapkyca** [idno/idno#667]
Especially new style multiple account syndication

**Comment by benwerd** [comment]
Yes. Required. We're actually talking about revamping the entire documentation site for easier access, addition, searching, and contribution. More soon.

**Comment by mapkyca** [comment]
Bump
